Directory.Build.props

```diff
@@ -0,0 +1,30 @@
+<Project>
+  <PropertyGroup>
+    <ComVisible>false</ComVisible>
+    <CLSCompliant>false</CLSCompliant>
+    <Platforms>AnyCPU;x64;ARM64;x86</Platforms>
+    <AnalysisLevel>preview-all</AnalysisLevel>
+    <LangVersion Condition="'$(TargetFramework)' == 'net48'">latest</LangVersion>
```

Comment:
Any reason for condition here?

Reply:
LangVersion is automatically determined by targeting net5.0, net6.0 etc. It is recommended to not explicitly set it.
For the classic .NET Framework this however defaults to C# 7.3 which would break the build.

Reply:
Dynamic language versions add some instability, so perhaps limiting it to 7.3 is a good way to go.
If you want to use C# 8.0 syntax on .Net Framework 4.8, you can restrict the version to 8.

Reply:
I'd say explicitly specify the latest available version at the moment for all frameworks.

Reply:
That's what I did originally but I changed it because of earlier comments.

Reply:
Hm which comment was that?

Reply:
https://github.com/CnCNet/xna-cncnet-client/pull/341#pullrequestreview-1083171448

I think it's fine if we leave it like this, if we remove .NET48 at some point and do a search for TargetFramework conditions throughout the codebase we'll see this one and remove it, as recommended by MS for .NET6.